A32NX/SimObjects/AirPlanes/Asobo_A320_NEO-LIVERY/aircraft.cfg

```diff
@@ -8,7 +8,7 @@ base_container = "..\Asobo_A320_NEO"
 ;===================== FLTSIM =====================
 
 [FLTSIM.0]
-title = "FlyByWire A320 NEO" ; Variation name
+title = "FlyByWire A320neo" ; Variation name
```

Comment:
If you're already at it, you could probably correct the name for the "default" A320 livery as well

Reply:
pushing the new fix right now!